nemesis/arena_battles.py

```diff
@@ -822,7 +822,7 @@ def item_dump():
 
 def quick_reset_logic():
     reset.reset_to_main_menu()
-    memory.ain.wait_frames(90)
+    memory.main.wait_frames(90)
```

Comment:
This is the only thing that hasn't been tested already. However, this change feels fairly straightforward.

Reply:
Barring a premature death or profound luck, this should be tested in an hour or two.

Reply:
This just got tested, it's working.